### PR TITLE
Issue 13: Extend description of if statements/expressions

### LINKED ISSUE #13: If with multi-line conditions
The style of if statements and if expressions is not consistent. Also `elsif` and `else` branches are not (always) considered.

### Current Style

Statement:

```Ada
if
   (((<condition>
     and then <condition>)
    or else <condition>)
   and then <condition>)
then
   <statements>
end if;
```

Expression:

```Ada
(if <condition>
 then <expression>
 else <expression>)
```

### Suggestion

Statement:

```Ada
if
   (((<condition>
     and then <condition>)
    or else <condition>)
   and then <condition>)
then
   <statements>
elsif
   (((<condition>
     and then <condition>)
    or else <condition>)
   and then <condition>)
then
   <statements>
else
   <statements>
end if;
```

Expression:

```Ada
(if
   (((<condition>
     and then <condition>)
    or else <condition>)
   and then <condition>)
 then
   <expression>
 elsif
   (((<condition>
     and then <condition>)
    or else <condition>)
   and then <condition>)
 then
   <expression>
 else
   <expression>)
```

### PR DESCRIPTION
**Pull request by treiher**
Close #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -187,15 +187,17 @@ function Bar (N : Natural) return Natural is
...
    <statements>
+elsif <condition> then
+   <statements>
+else
+   <statements>
 end if;
 ```
 
-With multi-line condition:
+If one of the `<condition>`s does not fit onto one line:
```
Comment (senier) [inline README.md]: I'd rephrase that to "If *any* of the `<condition>`s does not fit onto one line:"

File: README.md [hunk 2]
```diff
@@ -205,21 +207,37 @@ if
...
 
 ### If-expressions
 
 ```Ada
-(if <condition> then <expression> else <expression>)
+(if <condition> then <expression> elsif <condition> then <expression> else <expression>)
 ```
 
-If `<condition>` or one of the `<expression>`s do not fit onto one line:
+If the expression or one of the `<condition>`s or `<expression>`s do not fit onto one line:
```
Comment (senier) [inline README.md]: Why is expression mentioned twice? Maybe we should simplify that.

**Review by senier — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (treiher) [reply README.md]: What I tried to say here is: If the __whole if expression__ or one of __its__ `<condition>`s or `<expression>`s do not fit onto one line.

Comment (senier) [reply README.md]: That makes more sense. Thinking about it, wouldn't "if the whole expression does not fit onto one line" suffice?

Comment (treiher) [reply README.md]: Yes, I think you are right.

Comment (treiher) [reply README.md]: Fixed by 794c679.

File: README.md (hunk 1 above)
Comment (treiher) [reply README.md]: Fixed by 794c679.

**Review by jklmnn — APPROVED** [review]

**Review by senier — APPROVED** [review]
